app/models/register.rb

```diff
@@ -30,11 +30,12 @@ class Register < ApplicationRecord
       .count(:authority_id)
   }
   scope :available_count, -> { available.in_beta.count }
+  scope :category, -> { first.category }
```

Comment:
Is this needed, looks like it would work the same as that provided by the model association?

Reply:
It's now gone.